frontend/public/components/catalog-items.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
We should be using TypeScript for new components...

Reply:
arg..sorry, copied an existing file

Reply:
Could you also move all of the "catalog" components into a new directory?